src/tutorials/persistent_storage_tutorial.md

```diff
@@ -1,286 +1,529 @@
+Here are just a few ideas on ways persistent storage can be used:
 
-Really, anything that you might want to preserve for the next play session for that same player can be stored.
+* Player high score / player stats.
+* Player level / experience.
+* Player equipment / inventory.
+* Player resources (i.e. wood, metal, coal).
+* Currency (i.e. gold, gems).
+* Achievements / Trophies.
 
-![Overview Shot](../img/EditorManual/PersistentStorage/enablePlayerStorage.png){: .center loading="lazy" }
+--
 
-* **Completion Time:** 10 minutes
-* **Knowledge Level:** No knowledge *absolutely* required, but this will be easier to understand with a grasp on **[Lua](lua_basics_lightbulb.md)** already.
+* **Completion Time:** 20 minutes
+* **Knowledge Level:** No knowledge *absolutely* required, but this will be easier to understand with a grasp on **[Lua](../tutorials/lua_basics_lightbulb.md)** already.
 * **Skills you will learn:**
-    * How to store variables persistently between game sessions
-    * How to spawn stored template save data
+    * How to store data for a player persistently between game sessions.
+    * Sending the player score from the server to the client to display in the UI.
+    * How to spawn the last equipment the player used when they join.
 
 ---
 
-## Storage in Core Lua
+## Enable Player Storage
 
-Persistent Player Storage is available under the namespace called **Storage**. The available built-in Lua calls are:
+For your game to store data persistently for players, you will need to enable player storage. Doing this will allow you to use the [Storage API](../api/storage.md) to Set and Get data for players.
 
-* `Storage.GetPlayerData(Player)`
-    * Returns a table
-    * Is server-only
-* `Storage.SetPlayerData(Player, table)`
-    * Is server-only
+### Modify Game Settings
 
-All successfully stored data in preview mode can be viewed in your computer's File Explorer in `Saved/Maps/your_map_name/Temp/Storage/`. This data is just for debugging purposes and does not get uploaded to Core servers.
+1. In the **Hierarchy** window, look for a folder called **Gameplay Settings** and open that folder.
+2. Look for an object called **Game Settings** and click on it to make it the active object in the **Hierarchy**.
+3. In the **Properties** window, look for an option called **Enable Player Storage**.
+4. Click on **Enable Player Storage** so it is enabled.
 
-Each player table has a **maximum size limit of 32kb**.
+!!! tip "Missing Game Settings"
+    If you do not have a **Game Settings** object in your **Hierarchy**, it can added to your hierarchy from the **Core Content** window by searching for **Game Settings**.
 
-To read more about the supported data types that can be saved, as well as the possible error code results, check out the Storage section of the [Core API](../api/storage.md).
+![!Game Settings](../img/PersistentStorage/game_settings.png){: .center loading="lazy" }
 
----
+## Player Storage File Location
+
```

Comment:
maybe adding this section later when they do have storage from completing the tutorial steps could be useful to demonstrate. otherwise there wouldnt be a temp folder at this point.